src/components/Donate.vue

```diff
@@ -1,8 +1,12 @@
 <template>
   <ul>
-    <li class="c-crypto-item" v-for="crypto in cryptos" @click="copyToClipBoard(crypto.id)" :key="crypto.id">
+    <li class="c-crypto-item" v-for="(crypto, index) in cryptos" @click="copyToClipBoard(crypto.id, index)" :key="crypto.id">
       <label class="c-crypto-label">{{crypto.name}} ({{crypto.abreviation}})</label>
-      <div class="c-crypto-field">{{crypto.id}}</div>
+      <div class="c-crypto-field">
+        <span>{{crypto.id}}</span>
+
+        <div class="c-copied-feedback" :class="{ 'is-active': copied.id === crypto.id && copied.index === index }">Copied to clipboard!</div>
```

Comment:
Since `crypto.id` should be unique, and since we're already using it for the `:key`, I don't think this needs to compare the `.index` as well. We could probably remove all usage of `index` from this PR and things should still work, right?

Reply:
@taoeffect It was a while ago but I think I added that extra comparison because some `crypto` entry share the same `id` value, like you can see in the screenshot below.

<img src='https://github.com/okTurtles/groupincome.org/assets/17641213/6ff647ed-8bd3-480c-83f3-e69bd19cbc39' width='420'>

Btw, apparently your follow up request to update those payment addresses based on https://okturtles.org/donate/ will remove the need of that comparison.

Reply:
Yeah they should never have shared the same value... that's odd